src/views/TabView/TabBarBottom.js

```diff
@@ -39,6 +39,7 @@ type Props = {
   getLabelText: (scene: TabScene) => string;
   renderIcon: (scene: TabScene) => React.Element<*>;
   style: any;
+  labelStyle: any?;
```

Comment:
According to the docs about [any](https://flowtype.org/docs/quick-reference.html#any) it would also seem to me that the `?` is obsolute.  But I can't be sure.

Should I just remove the `?` here and in `TabBarTop` @ericvicenti ?